pcsx2/Config.h

```diff
@@ -17,6 +17,10 @@
 
 #include "x86emitter/tools.h"
 
+// Disable TAS on release builds, threading issue maybe? It hangs the emu.
+// Linux always hangs, Windows far less but it's still an issue.
+#define DISABLE_RECORDING
```

Comment:
With this PR hopefully addressing these hangs (still havn't seen any info on the Windows hangs) https://github.com/PCSX2/pcsx2/pull/3293 perhaps this could be avoided?